src/main/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -2044,35 +2044,7 @@ public int hashCode() {
             ret.addAll(errors);
             if (firstOnly) return ret;
         }
-        // TODO(mccowan): Is this asking "is this the primary alignment"?
```

Comment:
@mccowan !

Reply:
Thank you for cleaning up my loose ends!  😁. Hope all is well!